src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/FeatureSwitchManager.cs

```diff
@@ -221,7 +221,20 @@ public MethodIL GetMethodILWithInlinedSubstitutions(MethodIL method)
                     }
 
                     if ((flags[offset] & OpcodeFlags.EndBasicBlock) != 0)
+                    {
+                        if (reader.HasNext)
+                        {
+                            // If the bytes following this basic block are not reachable from anywhere,
+                            // the flags would make it look like the current instruction is just veeery long
```

Comment:
```suggestion
                            // the flags would make it look like the current instruction is just very long
```
I have read this as "veery" that is actually a bird name.